textile/chat-features.textile

```diff
@@ -144,6 +144,10 @@ h4(#rooms-lifecycle-operations). Room Lifecycle Operations
 *** @(CHA-RL7a1)@ The @RETRY@ operation - an internal process.
 *** @(CHA-RL7a2)@ The @RELEASE@ operation.
 *** @(CHA-RL7a3)@ The @ATTACH@ and @DETACH@ operations have equal precedence.
+* @(CHA-RL9)@ Many operations in the Chat SDK will still attempt to proceed if the room is in an @ATTACHING@ status. However, their ability to proceed will depend on the subsequent status that the room takes. This specification point is defined here to avoid unnecessary repetition. It is testable via the points that refer to it.
```

Comment:
Could we mark these three spec points as Testable? Motivation being that I want to write some tests for this shared behaviour, and want to be able to able to make use of [the Swift repo’s annotations for attributing tests to a spec point](https://github.com/ably-labs/ably-chat-swift/blob/main/CONTRIBUTING.md#attributing-tests-to-a-spec-point). But the Swift repo’s script for turning these annotations into a spec coverage report emits a warning if you have an annotation for a spec point that isn’t marked as Testable (the idea being to catch places where we've implemented spec points that were subsequently deleted).

tl;dr: it would help the Swift repo’s tooling if a-c were marked as Testable here, but if there's a strong reason for them not to be then I can work around it

Reply:
Yep, can certainly do that!